testing/person_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../person'
+
+describe Person do
+  describe '#initialize' do
+    it 'Creates a new person' do
+      person = Person.new(17, 'John Doe')
+      expect(person.name).to eq 'John Doe'
+    end
+  end
+end
+describe Person do
+  describe '#initialize' do
+    it 'Creates a new person' do
+      person = Person.new(17, 'John Doe')
+      expect(person.age).to eq 17
+    end
+  end
+end
```

Comment:
- [ ] Great job working on this milestone. We have a few more unit tests to add before we can proceed. Kindly add unit tests for 
- `can_use_services?` method. Add both `positive` and `negative` scenarios. See sample code snippet of positive scenario below:
 ```
   it "The can_use_services? method returns true if over age" do
      person = Person.new(19, "Sam", false)
      expect(person.can_use_services?).to be true
    end

```
- `add_rental` method
- `correct_name` method